draft/webxdc-dev-reference.md

```diff
@@ -54,12 +54,13 @@ and you won't get the update by `setUpdateListener()`.
 ### setUpdateListener()
 
 ```js
```

Comment:
```suggestion
```ts
```

Reply:
for a broader adaption, better stay with JavaScript and not TypeScript in this basic documentation.

there are far fewer ppl that can write TypeScript than JavaScript.

Reply:
` -> Promise<void>` is neither valid js to nor valid ts

Reply:
i'd go for `promise = window.webxdc.setUpdateListener((update) => {}, serial)` in the brief overview, and adapt the example with the `.then()` thingie.

Reply:
i created a little pr at https://github.com/deltachat/deltachat-core-rust/pull/3237